velox/docs/configs.rst

```diff
@@ -760,3 +760,74 @@ Tracing
      - integer
      - 0
      - The max trace bytes limit. Tracing is disabled if zero.
+
+Cache
+-----
+.. list-table::
+   :widths: 30 10 10 70
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - async-data-cache-enabled
+     - bool
+     - true
+     - If true, enable async data cache.
+   * - async-cache-ssd-gb
+     - integer
+     - 0
+     - The size of the SSD.
+   * - async-cache-ssd-path
+     - string
+     - /mnt/flash/async_cache.
+     - The directory that is mounted onto SSD.
+   * - async-cache-max-ssd-write-ratio
+     - double
+     - 0.7
+     - The max ratio of the number of in-memory cache entries being written to SSD cache over the total number of cache entries. This is to control SSD cache write rate, and once the ratio exceeds this threshold, then we stop writing to SSD cache.
+   * - async-cache-max-ssd-savable-ratio
+     - double
+     - 0.125
+     - The min ratio of SSD savable (in-memory) cache space over the total cache space. Once the ratio exceeds this limit, we start writing SSD savable cache entries into SSD cache.
+   * - async-cache-max-ssd-savable-bytes
+     - integer
+     - 1 << 24 (16 MB)
+     - Min SSD savable (in-memory) cache space to start writing SSD savable cache entries into SSD cache. NOTE: we only write to SSD cache when both above conditions are satisfied.
+   * - async-cache-persistence-interval
+     - string
+     - 0s
+     - The interval for persisting in-memory cache to SSD. Setting this config to a non-zero value will activate periodic cache persistence.
+   * - async-cache-ssd-disable-file-cow
+     - bool
+     - false
+     - In file systems, such as btrfs, supporting cow (copy on write), the ssd cache can use all ssd space and stop working. To prevent that, use this option to disable cow for cache files.
+   * - async-cache-ssd-disable-file-cow
+     - bool
+     - false
+     - In file systems, such as btrfs, supporting cow (copy on write), the ssd cache can use all ssd space and stop working. To prevent that, use this option to disable cow for cache files.
+   * - ssd-cache-checksum-enabled
+     - bool
+     - false
+     - When enabled, a CRC-based checksum is calculated for each cache entry written to SSD. The checksum is stored in the next checkpoint file.
+   * - ssd-cache-read-verification-enabled
+     - bool
+     - false
+     - When enabled, the checksum is recalculated and verified against the stored value when cache data is loaded from the SSD.
+   * - enable-serialized-page-checksum
```

Comment:
I don't think this config is for the SSD or async data cache. Instead, it is used for using checksum during the exchange of data pages.

Reply:
Removing this config